lib/seed_migration/register_entry.rb

```diff
@@ -12,14 +15,18 @@ def exclude(*attrs)
       attrs.map(&:to_s).each { |attr| exclude_single_attributes attr }
     end
 
-    def eql?(object)
-      object.class == self.class && object.model == self.model
+    def eql?(other)
+      other.class == self.class && other.model == self.model
     end
 
     def hash
       model.hash
     end
 
+    def order(order=DEFAULT_ORDER)
+      @order ||= order
+    end
```

Comment:
I'm not sure why this interface takes a default param. Would it make more sense to have

```
def order
    order || DEFAULT_ORDER
end
```